poetry.lock

```diff
@@ -1,21 +1,21 @@
 [[package]]
-name = "absl-py"
-version = "0.11.0"
-description = "Abseil Python Common Libraries, see https://github.com/abseil/abseil-py."
 category = "main"
+description = "Abseil Python Common Libraries, see https://github.com/abseil/abseil-py."
```

Comment:
why is there a change to `poetry.lock`? 🤔

Reply:
There was a merge conflict, I did `poetry lock` to regenerate the lock file.

Reply:
mhm, a merge conflict can only happen if you added `poetry.lock` to your changes in the first place, no? I'd suggest to remove that from the changeset then.